avdmanager/StatusBarManager.swift

```diff
@@ -315,11 +317,12 @@ struct ToolboxStyleEmulatorRow: View {
                 
                 // Action Buttons - Toolbox style
                 HStack(spacing: 8) {
-                    ToolboxActionButton(
-                        title: emulator.isRunning ? "Stop" : "Start",
-                        isRunning: emulator.isRunning,
-                        action: emulator.isRunning ? onStop : onStart
-                    )
+                    if (emulator.isRunning) {
```

Comment:
As per swift style guide 

```suggestion
                    if emulator.isRunning {
```